readme.md

```diff
@@ -34,26 +34,63 @@ A starter app is available on [codesandbox](https://codesandbox.io/s/github/matt
 ```shell
 npm install haunted
 ```
+
 For Internet Explorer 11, you'll need to use a proxy polyfill, in addition to the usual webcomponentsjs polyfills. 
+
 eg.
 ```html
-<script type="text/javascript" src="https://cdn.jsdelivr.net/npm/proxy-polyfill@0.3.0/proxy.min.js"></script> 
+<script src="https://cdn.jsdelivr.net/npm/proxy-polyfill@0.3.0/proxy.min.js"></script> 
 ```
-For a full example see - https://github.com/crisward/haunted-ie11
+
+For a full example with Internet Explorer 11, see - https://github.com/crisward/haunted-ie11
 
 You can also use Custom Elements without Shadow DOM if you wish.
 eg.
+
+```js
+component(() => html`...`, HTMLElement, { useShadowDOM: false }));
 ```
-component(() => html`...`, HTMLElement, {useShadowDOM: false}))
+
+### Importing
+
+__Haunted__ can be imported just like any other library when using a bundler of your choice:
```

Comment:
this advice is excellent, and will ring true after the implementation of the changes i'm requesting :+1: